data/battle_scripts_1.s

```diff
@@ -10011,7 +10007,7 @@ BattleScript_BerserkGeneRet_End:
 
 BattleScript_BoosterEnergyEnd2::
 	playanimation BS_SCRIPTING, B_ANIM_HELD_ITEM_EFFECT, sB_ANIM_ARG1
-	call BattleScript_AbilityPopUpTarget
+	call BattleScript_AbilityPopUpScripting
```

Comment:
I just realized that I wrote this. I deserve to be banished to the shadow realm for this.

Very smart.